var/spack/repos/builtin/packages/genesis/package.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Genesis(AutotoolsPackage, CudaPackage):
+    """GENESIS is a Molecular dynamics and modeling software
+    for bimolecular systems such as proteins, lipids, glycans,
+    and their complexes.
+    """
+
+    homepage = "https://www.r-ccs.riken.jp/labs/cbrt/"
+    url = "https://www.r-ccs.riken.jp/labs/cbrt/wp-content/uploads/2020/09/genesis-1.5.1.tar.bz2"
+    git = "https://github.com/genesis-release-r-ccs/genesis-2.0.git"
+
+    version("2.0b", branch="master")
```

Comment:
```suggestion
    version("master", branch="master")
```
You'll have to update this elsewhere too.

Reply:
Thank you for your advice . 
I fixed that according to your advice .